test/MarketMaker.js

```diff
@@ -344,45 +328,37 @@ contract('marketMaker', function(accounts) {
     }
     assert.equal(true, thrown, "claimEth should have thrown");
 
-    try {
-      thrown = false;
-      await claimEth(0.03);
```

Comment:
why was this block removed? We were asserting that claiming more than what's allowed should throw an exception, now we are not checking that

Reply:
Because it was checking it twice, its checked above.

Reply:
ok